micrometer-core/src/main/java/io/micrometer/core/instrument/Observer.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2017 Pivotal Software, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.micrometer.core.instrument;
+
+/**
+ * @author Dmitry Poluyanov
+ * @since 23.07.17
+ */
+public interface Observer {
```

Comment:
My bad, completely forgot about aggregation quantiles through all instances. Yes, `Observer` originally was planned as abstraction for different summaries meter without affecting current micrometer `Quantile` and `Histogram` design.

And later i want to experiment with storing large amount of measurements-per-step (possible off-heap) in one metric (e.g. not-aggregated timings for every HTTP request) for sends it to backend and use storage-side quantiles calculation. 
Think that i can do it with my own `Meter` and `Quantile` implementation and there are no need in common `Observer` abstraction. I will update this PR after my researches.